src/ui/index.jsx

```diff
@@ -141,6 +141,7 @@ ipcRenderer.on('open-uri-requested', (event, uri, newSession) => {
     } else if (isURIValid(uri)) {
       const formattedUri = formatLbryUriForWeb(uri);
       app.store.dispatch(push(formattedUri));
+      analytics.openUrlEvent(formattedUri);
```

Comment:
@kauffj Not sure if this should be `formattedUri` which is something like `/one`, or `uri` which is `Lbry://one` (GA makes the first letter uppercase)

Reply:
Ambivalent, `formattedUri` is probably fine.